### DISCUSSION TITLE
Best way to use Maizzle without Tailwind

### DISCUSSION THREAD

**Discussion by JPBetley** [maizzle/framework#567] [Q&A] [answered]
I have a specific project I wish to use Maizzle for, but not with Tailwind. Right now I have removed the tailwind imports in the compiled css file, but was wondering if that's the best way to go about doing this. So far, it's only outputting expected CSS, but I wanted to see if there's a better way

**Comment by cossssmin** [comment, marked as answer]
There isn't a way to prevent Maizzle from compiling Tailwind if that's what you're asking for.

Even if you don't have a `tailwind.config.js` file, the [tailwind generator](https://github.com/maizzle/framework/blob/master/src/generators/tailwindcss.js) will still compile Tailwind with a default config and a bunch of PostCSS plugins, but speed-wise it's not really an issue if you leave JIT mode enabled in Tailwind, as that is very fast and you'll most likely not feel it running. Besides that, simply don't use any Tailwind classes and that's all there is to it 👍
